### PR TITLE
Test upload part 2

### PR DESCRIPTION
**Pull request by BrandonWeart**

### CODE REVIEW

File: et --soft 9b4b79f2e9236ea5c785e7d5d26abf996b685b6a
Comment (zssherman) [inline et --soft 9b4b79f2e9236ea5c785e7d5d26abf996b685b6a:1]: I would remove any git log file etc from the git push.

Comment (zssherman) [reply et --soft 9b4b79f2e9236ea5c785e7d5d26abf996b685b6a:1]: Which could be part of the test! Can try a git rm on it and git commit and git push to remove it.

Comment (BrandonWeart) [reply et --soft 9b4b79f2e9236ea5c785e7d5d26abf996b685b6a:1]: Yeah Ill do that

Comment (BrandonWeart) [reply et --soft 9b4b79f2e9236ea5c785e7d5d26abf996b685b6a:1]: Okay should be good

**Pull request closed without merging**